src/configuration.ts

```diff
@@ -503,6 +503,11 @@ export interface IChromiumLaunchConfiguration extends IChromiumBaseConfiguration
    * If true, will start profiling soon as the page launches.
    */
   profileStartup: boolean;
+
+  /**
+   * Close whole browser or just the tab when cleaning up
+   */
+  cleanUp: 'wholeBrowser' | 'onlyTab' | null;
```

Comment:
I don't think null is a valid value here

Reply:
Removed